.circleci/config.yml

```
@@ -51,17 +51,25 @@ workflows:
                 - lts
                 - maintenance
       - release-management/test-nut:
-          matrix:
-            parameters:
-              os:
-                - linux
-                - windows
-              sfdx_version:
-                - latest
-              node_version:
-                - latest
+          name: nuts-on-linux
+          sfdx_version: latest
+          requires:
+            - release-management/test-package
+      - release-management/test-nut:
+          name: nuts-on-windows
+          node_version: lts
+          sfdx_version: latest
+          os: windows
           requires:
             - release-management/test-package
+      - release-management/run-command-keyword:
+          name: sandbox-nuts
+          git_key_word: '[sb-nuts]'
+          timeout: 1.5h
+          command: test:nuts:sandbox
+          requires:
+            - nuts-on-windows
+            - nuts-on-linux
```

If we leave the matrix up above this can just be changed to `release-management/test-nut`
```suggestion
            - release-management/test-nut
```